website_sale_product_minimal_price/models/product_template.py

```diff
@@ -24,18 +24,35 @@ def _get_combination_info(
         """
         Update product template prices for products items view in website shop
         render with cheaper variant prices.
+        Checking the context, we can know if we are on grid view or at the product view.
+        With this comprobation we avoid problems rendering the price at the product
+        view.
         """
+        if pricelist and "bin_size" in self.env.context:
+            """
+            In this version the categories are not contempled.
+            """
+            pricelist_items = pricelist.item_ids.filtered(
+                lambda i: i.product_tmpl_id.id == self.id
+                or i.product_id.product_tmpl_id.id == self.id
+                and i.min_quantity > add_qty
+                and (not i.date_start or i.date_start <= fields.Date.today())
+                and (not i.date_end or fields.Date.today() <= i.date_end))
```

Comment:
Don't need invoke 'today()' all the times.
```suggestion
            date_now = fields.Date.today()
            pricelist_items = pricelist.item_ids.filtered(
                lambda i: i.product_tmpl_id.id == self.id
                or i.product_id.product_tmpl_id.id == self.id
                and i.min_quantity > add_qty
                and (not i.date_start or i.date_start <= date_now)
                and (not i.date_end or date_now <= i.date_end))
```